docs/website/docs/user-guides/deploy/_running_deploy.mdx

```diff
@@ -0,0 +1,42 @@
+import CodeBlock from '@theme/CodeBlock';
+import Version from '../../_version.mdx';
+
+Now we're ready to run `odo deploy`:
+
+```console
+odo deploy
+```
+
+<div>
+<CodeBlock language="console">
+{`\
+$ odo deploy
+  __
+ /  \\__     Deploying the application using `}{props.name}{` Devfile
+ \\__/  \\    Namespace: default`}<Version/>{`
+↪ Building & Pushing Container: MYUSERNAME/`}{props.name}{`-odo-example
+ •  Building image locally  ...
+ ✓  Building image locally [880ms]
+ •  Pushing image to container registry  ...
+ ✓  Pushing image to container registry [5s]
+\n
+↪ Deploying Kubernetes Component: `}{props.name}{`-example
+ ✓  Searching resource in cluster
+ ✓  Creating kind Deployment [48ms]
+\n
+↪ Deploying Kubernetes Component: `}{props.name}{`-example
+ ✓  Searching resource in cluster
+ ✓  Creating kind Service [51ms]
+\n
+↪ Deploying Kubernetes Component: `}{props.name}{`-example
+ ✓  Searching resource in cluster
+ ✓  Creating kind Ingress [49ms]
+\n
+Your Devfile has been successfully deployed
+`}
+</CodeBlock>
+</div>
+
+Your application has now been deployed to the Kubernetes cluster with Deployment, Service, and Ingress resources.
```

Comment:
Maybe we could also mention that the user might need to change the image visibility to Public in the container registry. New images pushed to the registry might be private by default.

Reply:
IMO this would be a cluster problem, no? Thoughts @kadel

Reply:
I think this is a registry specific issue. If you are using Quay, you need to be aware of changing visibility. Not so for Docker Hub.

Reply:
> I think this is a registry specific issue. If you are using Quay, you need to be aware of changing visibility. Not so for Docker Hub.

Yes, on Docker Hub, I think it depends on the settings at the Organization level, if any. But by default, on personal accounts, there is 1 private repository and any new one is public by default.

Indeed, I was thinking of just mentioning here that depending on the registry they pushed to, users might need to change the image visibility. Otherwise, the application might not be reachable and users might spend time figuring out why.

Reply:
I don't think we should include this in the PR.

This is just reorganizing and not adding new content.

I've created an issue for this here: https://github.com/redhat-developer/odo/issues/6107